pkg/api/openapi/minder/v1/minder.swagger.json

```
@@ -2396,6 +2403,57 @@
           "UserService"
         ]
       }
+    },
+    "/api/v1/user/invitation/{nonce}": {
```

```suggestion
    "/api/v1/user/invitation/{invite}": {
```

my understanding that a more specific name for the nonce could be "invitation code" or "invite" or "invite code".

---

Let's call it `code` here, so that we don't conflate it with e.g. a database record.